### PR TITLE
Add support for kdyby/doctrine-cache

### PR DESCRIPTION
**Pull request by enumag**
This is what I came up with when trying to solve #15.

Don't merge yet, I need to test it first.

### CODE REVIEW

File: src/Kdyby/Validator/DI/ValidatorExtension.php [hunk 1]
```diff
@@ -52,16 +54,18 @@ public function loadConfiguration()
...
 		$cacheFactory = self::filterArgs($config['cache']);
-		if (!class_exists($cacheFactory[0]->entity) || !in_array('Symfony\Component\Validator\Mapping\Cache\CacheInterface', class_implements($cacheFactory[0]->entity), TRUE)) {
-			throw new Nette\Utils\AssertionException(
-				'Expected implementation of Symfony\Component\Validator\Mapping\Cache\CacheInterface, ' .
-				'but ' . $cacheFactory[0]->entity  . ' given.'
-			);
+		if (class_exists($cacheFactory[0]->entity) && in_array('Symfony\Component\Validator\Mapping\Cache\CacheInterface', class_implements($cacheFactory[0]->entity), TRUE)) {
+			$cacheService->setFactory($cacheFactory[0]->entity, $cacheFactory[0]->arguments);
+		} else {
+			$cacheService->setFactory('Symfony\Component\Validator\Mapping\Cache\DoctrineCache', array(
```
Comment (fprochazka) [inline src/Kdyby/Validator/DI/ValidatorExtension.php]: I would rather implement a bridge for symfony cache like doctrine-cache is, but without the actuall doctrine

Comment (enumag) [inline src/Kdyby/Validator/DI/ValidatorExtension.php]: Symfony itself uses doctrine/cache for validator.

Comment (fprochazka) [inline src/Kdyby/Validator/DI/ValidatorExtension.php]: Well, that I either didn't know or forgot :)

**Comment by enumag** [comment]
@fprochazka Since https://github.com/Kdyby/DoctrineCache/pull/7 is merged we should probably merge this as well for now.

**Comment by fprochazka** [comment]
I've restarted the build and it's green. It doesn't say much, considering we have only one test, but I trust you :)

**Comment by fprochazka** [comment]
@enumag thank you

**Pull request merged**